packages/gatsby-theme-newrelic/src/utils/createTessen.js

```diff
@@ -58,26 +58,28 @@ const tessenAction =
     const customerId = JSON.parse(Cookies.get('ajs_user_id') || 'null');
     const anonymousId = JSON.parse(Cookies.get('ajs_anonymous_id') || 'null');
 
-    window.Tessen[action](
-      eventName,
-      {
-        ...properties,
-        env: config.env || '',
-        category,
-        nr_product: config.product,
-        nr_subproduct: config.subproduct,
-        location: 'Public',
-        customer_user_id: customerId,
-        anonymousId,
-      },
-      {
-        Segment: {
-          integrations: {
-            All: true,
-          },
+    if (window.Tessen) {
```

Comment:
Looks like this is already covered on line 51-56:

```js

    if (!window.Tessen) {
      return warning(
        false,
        `tessen.${eventName}: You are attempting to use a Tessen action, but Tessen is not available on 'window'. Calls to '${eventName}' will result in a noop.`
      );
    }
```

Reply:
good catch 🙈